test/contract-mock.js

```diff
@@ -1,19 +1,18 @@
 const ethUtil = require('ethereumjs-util');
+const utils = require('./utils');
 
 const erc725CoreInterfaceID = '0xd202158d';
 const erc725InterfaceID = '0xdc3d2a7b';
 
 module.exports = class MockContract {
   constructor(options) {
-    this.isSupportsERC725CoreInterface = options.isSupportsERC725CoreInterface;
-    this.isSupportsERC725Interface = options.isSupportsERC725Interface;
-    this.actionableKey = options.actionableKey;
-    this.errorOnIsSupportedContract = options.errorOnIsSupportedContract;
-    this.errorOnKeyHasPurpose = options.errorOnKeyHasPurpose;
+    this.authorizedKey = options.authorizedKey;
+    this.address = options.address;
+    this.errorIsValidSignature = options.errorIsValidSignature;
```

Comment:
Sorry, having a hard time with this - what is `errorIsValidSignature` used for?

Reply:
This is used so the test can manually tell the wallet: "when `IsValidSignature` is called, please error out`"

Reply:
Ahh, I see. Thanks 👍